src/os/shared/osapi-module.c

```diff
@@ -241,7 +241,11 @@ int32 OS_ModuleLoad ( uint32 *module_id, const char *module_name, const char *fi
         record->name_entry = OS_module_table[local_id].module_name;
 
         /* Now call the OS-specific implementation.  This reads info from the module table. */
+        /// TODO-MAC: Error handling was missing.
```

Comment:
This has nothing to do with running on macOS. However it caused me additional troubles when debugging and it is an important thing to have (full error handling). I will sign the individual CLA now and will open this one as a separate PR.